tests/libtest.sh

```diff
@@ -79,7 +86,7 @@ cleanup() {
   # cases it has been observed that udev rule kept the device
   # busy.
   udevadm settle
-  clean_config_files
+  clean_config_files $INFILE $OUTFILE
```

Comment:
I think you have lost some changes and gone back to old code. So it needs to be redone.